ios-base/DroidKaigi 2020/Session/Models/Session.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Session.swift
+//  DroidKaigi 2020
+//
+//  Created by 伊藤凌也 on 2020/01/18.
+//
+
+import api
+
+struct Session {
```

Comment:
So sorry. There is ResponseToModelMapper.kt and probably you can use it 🙇 
So you can use Model class🙇 
Can you try it 🙏

Reply:
I think it was really hard. So sorry 🙇